docs/nodes/validate/add-a-validator.md

```diff
@@ -74,156 +75,332 @@ Once your validator’s start time has passed, you will see the rewards it may e
+`delegationFeeRate`
+
+Avalanche allows for delegation of stake. This parameter is the percent fee this validator charges when others delegate stake to them. For example, if `delegationFeeRate` is `10` and someone delegates to this validator, then when the delegation period is over, 10% of the reward goes to the validator and the rest goes to the delegator.
+
+#### Stake Amount
+
+In order to validate the FUJI Network, one must stake AVAX.
+
+The sample code assigns the variable, `stakeAmount` to be the minimum amount of AVAX that a node must stake to become a validator(1 AVAX).
+```ts
+const main = async (): Promise<any> => {
+  const stakeAmount: any = await pchain.getMinStake()
+  ...
+}
+```
+
+To stake a different amount, please replace the code above with a stake amount of 1 AVAX or greater. Below sets a new value of 10,000 AVAX (`10000000000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX. 
+
+Please follow the Example below for steps to change the stake amount:
+
+Locate this line in the file
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    stakeAmount.minValidatorStake,
+    ...
+  )
+```
+
+and change the default value to a new `BN` value, for example:
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    new BN(1e12),
+    ...
+  )
+```
+
+
+#### Start and End Time
+
+The sample code uses `const startTime: BN = UnixNow().add(new BN(60 * 1))` and
+`const endTime: BN = startTime.add(new BN(26300000))` to compute the Unix time 1 minute and 304 days in the future to use as the values of `startTime` and `endTime`, respectively.
+
+:::tip
+You can create your own unix timestamp [here](https://www.unixtimestamp.com/) or by using the `UnixNow()` method
+:::
+
+To create your own start times, please follow the steps below:
+
+1. Locate this line in the file
+
+```ts
+const startTime: BN = UnixNow().add(new BN(60 * 1))
+const endTime: BN = startTime.add(new BN(26300000))
+```
+
+2. Change `startTime` and `endTime` to new `BN` values, for example:
+
+```ts
+const startTime: BN = new BN(1654656829) // Wed Jun 08 2022 02:53:49 GMT+0000
+const endTime: BN = new BN(1662602029) // Thu Sep 08 2022 01:53:49 GMT+0000
+```
+
+#### Addresses
 
-In order to validate the Primary Network, one must stake AVAX. This parameter defines the amount of AVAX staked.
+`toAddresses`
 
-`rewardAddress`
+An array of addresses as [Buffer](https://github.com/feross/buffer) who receive the staked tokens at the end of the staking period.
 
-When a validator stops validating the Primary Network, they will receive a reward if they are sufficiently responsive and correct while they validated the Primary Network. These tokens are sent to `rewardAddress`. The original stake will be sent back to an address controlled by `username`.
+`fromAddresses`
+
+An array of addresses as a [Buffer](https://github.com/feross/buffer) who own the staking UTXOs.
+
+`changeAddresses`
```

Comment:
Are `change`, `to` and `reward` really arrays? On the platform.addValidator() those are all single address, not an array of them.

Reply:
This was pulled from `avalanchejs/avalanche/dist/apis/platformvm/api.d.ts` 

```js
     * @param toAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who received the staked tokens at the end of the staking period
     * @param fromAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who own the staking UTXOs the fees in AVAX
     * @param changeAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who gets the change leftover from the fee payment
     * @param rewardAddresses The addresses which will receive the rewards from the delegated stake.
```

Also, I tried to pass a single string as an argument [here](https://github.com/ava-labs/avalanchejs/blob/ffaf4d2ef2bbc62a57685d611e3a98115a2d2195/examples/platformvm/buildAddValidatorTx.ts#L41-L50) but the compiler asked for a string[]

Could use clarity on this